docs/README.md

```diff
@@ -7,4 +7,20 @@
    * Defines the data payloads to enable interoperability between `kfserving` components.
 
 # Architecture Overview
+The KFService Data Plane architecture consists of a static graph of components. The diagram may not be up to date with the current set of features available in the graph, but the overall mechanism holds:
+
+- The User defines the components they wish to use in a KFService.
+  - The only required component is a Predictor.
+  - Additional components may be specified to attach addtional behavior. 
+  - Each component is specified at the ModelSpec layer, allowing for canarying.
+- The KFService will support one of several Data Plane interfaces.
+  - For example: cloud-ai-platform/HTTP, cloud-ai-platform/GRPC, seldon/HTTP, seldon/GRPC
+  - Data Plane interfaces may be specified with annotations and defaults to cloud-ai-platform/HTTP
```

Comment:
Why default to cloud-ai-platform/HTTP? It should default to an open source alternative?

Reply:
This is basically the tensorflow/HTTP protocol. I call it cloud-ai-platform because we *slightly* decorate this API to include explainability. I think I'll go back to calling it tensorflow/HTTP ;)